### ISSUE TITLE
Minor: Form Label

### ISSUE THREAD

**Issue by magalj** [INFO442-Team2/Seattle-Sustenance#18]
Minor
Description:
If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.
Reproduction:
1.	F12 developer view on home page / sign in
2.	Select input elements
3.	<label for="input element name"> </label> in <form></form> not present

**Comment by rsgoertz** [comment]
Ran through the form using NVDA for Windows. It appears to pass all tests and properly addresses the form and inputs. 

Screen Reader Transcript:

Seattle Sustenance  document
search by zip code...  edit  search by zip code...
Seattle Sustenance  document
search by zip code...  edit  search by zip code...
search by name...  edit  search by name...
